### PR TITLE
tail -[num]f option

### PR DESCRIPTION
**Pull request by eliotsykes**

### CODE REVIEW

**Comment by igorshubovych** [comment]
LGTM

**Comment by waldyrious** [comment]
this is the same as `-100 -f`, right? I'd prefer not to introduce notational quirks that may confuse beginners.

**Comment by igorshubovych** [comment]
@waldyrious, in your opinion should it be splitted then?

We have `-n {{num}}`, but `-{{num}}` is even simpler.

**Comment by waldyrious** [comment]
Simpler, yes, but less intuitive for beginners. If we want we can introduce the shortened notation, but I'd prefer doing that separately from the trick of gluing two options together.

**Comment by igorshubovych** [comment]
Well said. Anyway thank you for your contribution @eliotsykes.

**Comment by ostera** [comment]
I think the text could be rephrased. I find it easier to think of `tail -f` as `stream a file` than `keep reading till ctrl-c`, but then again I usually just think of `tailing` a file. 

Other than that, we should try to keep parameters separated. `-f`, `-n`, `-r` instead of mashing them up.

**Comment by eliotsykes** [comment]
Thanks for all the feedback. 

I'm don't know if any of these are appropriate names, but here goes...

Is an "in-practice"/"common-vernacular"/"pro-tip"/"in-the-wild" section for any tldr pages out of scope? This could list these kind of sample/combined usages that occur in the wild.

**Comment by igorshubovych** [comment]
TLDR was created with different purpose: provide simplified man-pages. So it is for cases when you remember the command but do not remember the key.

There are other projects with different purpose: [cheat](https://github.com/chrisallenlane/cheat) is great, and its its is to provide this quick advanced tips. There are good cheatsheets in Dash@OSX.

**Comment by eliotsykes** [comment]
Thanks @igorshubovych - checking out cheat now, much appreciated! Closing this PR down.
